src/ontology/mondo.Makefile

```diff
@@ -1298,6 +1302,24 @@ update-subclass-sync: $(TMPDIR)/subclass-confirmed.robot.owl tmp/subclass-axioms
 		make NORM
 		mv NORM $(SRC)
 
+tmp/synonyms-axioms.owl: $(SRC)
+	$(ROBOT) filter --input $(SRC) --term oboInOwl:hasExactSynonym --axioms annotation --preserve-structure false --trim false \
+		--drop-axiom-annotations "oboInOwl:source=~'(DOID|ICD10CM|icd11.foundation|NCIT|OMIM|OMIMPS|Orphanet):.*'" \
```

Comment:
Never seen this syntax before!

Guess there's no easy way to DRYify this and interpolate these prefixes from elsewhere.

Reply:
Yes this is a new ROBOT feature..

I would have liked to dryify this as well but I cant see an easy way right now.. somewhere there would have to live a source of truth for the resources that we want to sync with in this pipeline, so I thought to myself, might as well be here, but yes, its not DRY

Reply:
@matentzn IC10WHO is missing here. Not sure if anything else, but you should double check the prefix_maps in the metadata files in `mondo-ingest` if you want to double check for any other missing ones.

RE: DRY: 1st step: Maybe could then add these sources to a global variable and interpolate it? Not too important at this point.

Reply:
> RE: DRY: 1st step: Maybe could then add these sources to a global variable and interpolate it? Not too important at this point.

Yeah, good point, did that! Immediately refactored the respective code for the subclass sync as well. Definetely DRYer!